docs/Getting Started/Installation/index.md

```diff
@@ -3,7 +3,24 @@ In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software dev
 
 ## Install SDL SDK
 @![iOS]
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
+
+### Accio
+You can install this library using [Accio](https://github.com/JamitLabs/Accio), which is based on SwiftPM syntax. Please follow the steps on the Accio README linked above to initialize Accio into your application. Once installed and initialized into your Xcode project, the root directory should contain a Package.swift file.
+
+1\. Open the Package.swift file.
+
+2\. Add the following line to the dependencies array of your package file. We suggest always using the latest release of the SDL library. 
+
+    ```.package(url: "https://github.com/smartdevicelink/sdl_ios.git", .upToNextMajor(from: "6.2.0"),```
```

Comment:
```suggestion
```swift
.package(url: "https://github.com/smartdevicelink/sdl_ios.git", .upToNextMajor(from: "6.2.0"),
```
```

Reply:
I'm not sure that this rendered correctly